### PR TITLE
Cone actor uses `repeat_primitive` by default

### PR DESCRIPTION
**Pull request by xtanion**
Resolves issue #529

### CODE REVIEW

File: fury/actor.py [hunk 1]
```diff
@@ -1858,6 +1858,8 @@ def cone(centers, directions, colors, heights=1., resolution=10,
         If faces is None then a cone is created based on directions, heights
         and resolution. If not then a cone is created with the provided
         vertices and faces.
+    use_primitive: boolean
```
Comment (skoudoro) [inline fury/actor.py:1861]: `bool, optional` instead

**Review by skoudoro — COMMENTED** [review]
Hi @xtanion,

There is some conflict in your branch. Can you resolve them? 
Can you update the tutorial or add one.

look at this [PR](https://github.com/fury-gl/fury/pull/552/files) for an example

Thank you

File: fury/primitive.py [hunk 2]
```diff
@@ -906,3 +906,69 @@ def prim_cylinder(radius=0.5, height=1, sectors=36, capped=True):
...
 
     return vertices, triangles
+
+
+def prim_cone(radius=0.5, height=1, sectors=10):
+    """Return vertices and triangle of a Cone.
+
+    Parameters
+    ----------
+    radius: float
```
Comment (skoudoro) [inline fury/primitive.py:916]: `float, optional` instead

File: fury/primitive.py [hunk 3]
```diff
@@ -906,3 +906,69 @@ def prim_cylinder(radius=0.5, height=1, sectors=36, capped=True):
...
+        Radius of the cone
+    height: float
```
Comment (skoudoro) [inline fury/primitive.py:918]: `float, optional` instead

File: fury/primitive.py [hunk 4]
```diff
@@ -906,3 +906,69 @@ def prim_cylinder(radius=0.5, height=1, sectors=36, capped=True):
...
+        Height of the cone
+    sectors: int
```
Comment (skoudoro) [inline fury/primitive.py:920]: `int, optional` instead

File: fury/primitive.py [hunk 5]
```diff
@@ -906,3 +906,69 @@ def prim_cylinder(radius=0.5, height=1, sectors=36, capped=True):
...
+    Returns
+    -------
+    vertices: ndarray
+        vertices coords that compose our cone
+    triangles: ndarray
+        triangles that compose our cone
+
+    """
+
+    if not sectors > 2:
```
[the remark this answers is not in the document]
Comment (xtanion) [reply fury/primitive.py]: > Hi @xtanion,
> 
> There is some conflict in your branch. Can you resolve them? Can you update the tutorial or add one.
> 
> look at this [PR](https://github.com/fury-gl/fury/pull/552/files) for an example
> 
> Thank you

Thanks for the review @skoudoro, I made the changes, please take a look.
Thanks.

[the remark this answers is not in the document]
Comment (skoudoro) [reply fury/primitive.py]: Thank you for the update. Unfortunately, tests are failing so I will check only after you fix the tests.

[the remark this answers is not in the document]
Comment (xtanion) [reply fury/primitive.py]: Hi @skoudoro, I just figured out that `advanced_geometry_test` was failing due to the `arrow_actor`. changing the `height` parameter with `scale` fixes the size of arrows 😅  .
I changed the `axes` actor's `repeat_primitive=False`, `window.snapshot` wasn't detecting colors while using primitive arrows.

**Review by skoudoro — COMMENTED** [review]
Thank you for the update. See below for additional comments.

File: docs/tutorials/01_introductory/viz_cone.py [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+scene.add(cone_actor1)
+scene.add(cone_actor2)
+
+interactive = True
+
+if interactive:
+    window.show(scene, size=(600, 600))
+
+window.record(scene, out_path='viz_markers.png', size=(600, 600))
```
[the remark this answers is not in the document]
Comment (xtanion) [reply docs/tutorials/01_introductory/viz_cone.py]: Wrote that by mistake, Modified to `viz_cone`.

File: docs/tutorials/01_introductory/viz_cone.py [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+scene = window.Scene()
+
+############################################################################
+# Adding our cone actors to scene.
+
+scene.add(cone_actor1)
+scene.add(cone_actor2)
+
+interactive = True
```
[the remark this answers is not in the document]
Comment (xtanion) [reply docs/tutorials/01_introductory/viz_cone.py]: Done

**Comment by skoudoro** [comment]
Thank you @xtanion for this

**Pull request merged**